piyushgarg-dev/piyushgargdev-nextjs
# fix:  #130 socialmedia icons color on hover

## What does this PR do?
social media icons color is changing on hover in Navbar. i used color picker to make the exact same colors as the original websites.

Fixes #130 

![Screenshot (35)](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/92119614/ec10a44e-c6f7-47a4-878f-9cdc09dadc1c)


## Type of change



- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)


## How should this be tested?
you can visit on homepage and check the social media icons color by hovering it.

- [X ] Test A
